pyteal/ast/txn.py

```diff
@@ -110,6 +110,22 @@ class TxnField(Enum):
     created_application_id = (61, "CreatedApplicationID", TealType.uint64, False, 5)
     last_log = (62, "LastLog", TealType.bytes, False, 6)
     state_proof_pk = (63, "StateProofPK", TealType.bytes, False, 6)
+    approval_program_pages = (64, "ApprovalProgramPages", TealType.bytes, True, 7)
```

Comment:
@barnjamin Minor:  Can you extend https://pyteal.readthedocs.io/en/stable/accessing_transaction_field.html to cover the new fields?

Based on the existing grouping, I'm thinking https://pyteal.readthedocs.io/en/stable/accessing_transaction_field.html#application-call is the best fit.